app/views/super_admin/whitelists/email_domains/index.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :page_title, "Manage access - GovWifi" %>
```

Comment:
This doesn't quite sound right to me. Something like "Manage allowed domains - GovWifi" is more accurate.